paver.sh

```diff
@@ -552,6 +552,12 @@ detect_os() {
                 check_os_version "SUSE Linux Enterprise Server" 10 \
                     "$os_version_raw" os_version_chevah
                 OS="sles${os_version_chevah}"
+                if [ ${os_version_chevah} -eq 11 ]; then
+                    # Check for SLES 11 with OpenSSL 1.0 (aka Security Module).
+                    if [ -n $(rpm -q openssl1) ]; then
```

Comment:
well. paver.sh is used outside of the python-package so we are pushing this dependency to all places where paver.sh is used on sles